### PR TITLE
Create first draft of the OpenAPI spec for the PPR API #150

### PR DESCRIPTION
**Pull request by jguertin**
This file was drafted with [Stoplight Studio](https://stoplight.io/) and provides an initial draft for implementing search functionality reflecting discussion from the other day.

Nothing here is set in stone, so I'm happy to make any changes, and would suggest it remain quite flexible as we implement functionality.

#150

### CODE REVIEW

**Review by WalterMoar — APPROVED** [review]
(can you connect this PR to the issue?)

I missed the reasoning on why we're doing this, as opposed to just stubbing it out in FastAPI. Looks good, though.

File: ppr-api/docs/api_contract/ppr-api.v1.yaml [hunk 1]
```diff
@@ -0,0 +1,313 @@
+openapi: 3.0.0
+info:
+  title: Personal Property Registry API
+  version: '1.0'
+  description: PPR API Specs
+  contact:
+    name: BC Registries
+servers:
+  - url: 'http://localhost:3100'
```
Comment (bryan-gilbert) [inline ppr-api/docs/api_contract/ppr-api.v1.yaml:9]: port 3100?

File: ppr-api/docs/api_contract/ppr-api.v1.yaml [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+        ref:
+          type: string
+          format: uri
+        type:
+          type: string
+          enum:
+            - SECURITY_AGREEMENT
+            - REPAIRERS_LIEN
+        registeringParty:
+          type: object
```
Comment (bryan-gilbert) [inline ppr-api/docs/api_contract/ppr-api.v1.yaml:213]: Should this be array as a FS can have more than one registering party

**Review by bryan-gilbert — APPROVED** [review]
Non-blocking comments.  Good to be making progress.

**Comment by jguertin** [comment]
> (can you connect this PR to the issue?)
> 
> I missed the reasoning on why we're doing this, as opposed to just stubbing it out in FastAPI. Looks good, though.

This is intended as design documentation for the API Contract to inform the implementation of upcoming stories.  It captures discussion we had last week and will serve to provide common understanding as we develop features and interfaces.

With regards to location in the code base, I followed what was modelled in the lear and auth projects.  I recommend using Stoplight Studio to review this file.

#105

**Pull request merged**